examples/kubernetes/templates/v1/cilium-ds.yaml.sed

```diff
@@ -57,6 +57,7 @@ spec:
         args:
           - "--debug=$(CILIUM_DEBUG)"
           - "-t=vxlan"
+          - "--k8s-legacy-host-allows-world=$(CILIUM_LEGACY_HOST_ALLOWS_WORLD)"
```

Comment:
You can remove this option configure viper to use the value from the environment variable of `CILIUM_LEGACY_HOST_ALLOWS_WORLD`